docs/extended_api/synchronization_library/pipeline/producer_acquire.md

```diff
@@ -0,0 +1,17 @@
+---
+nav_exclude: true
+---
+
+# cuda::pipeline\<Scope>::**producer_acquire**
+
+```c++
+void producer_acquire();
+```
+
+Blocks the current thread until the next _pipeline stage_ is available.

```

Comment:
Do we document the relationship between `producer_acquire` and `cuda::pipeline_shared_state`'s `StagesCount` template parameter somewhere?

If not, I think we should document that somewhere.